pkg/virt-api/rest/subresource.go

```diff
@@ -680,7 +680,7 @@ func (app *SubresourceAPIApp) StopVMRequestHandler(request *restful.Request, res
 		if err != nil {
 			return
 		}
-	case v1.RunStrategyManual:
+	case v1.RunStrategyRerunOnFailure, v1.RunStrategyManual:
```

Comment:
We should also adjust method comment:
https://github.com/kubevirt/kubevirt/pull/10639/commits/18044cc14dd9732f178b1e92befa7f5e73564a1f#diff-028d148aa715883f3fd17ae07f8a03225251bdd5eb1893566dfb68085298a434L608-L612